pages/docs/ecosystem-roles/validator/requirements.mdx

```diff
@@ -123,7 +123,63 @@ cargo build --release
 > NOTE: You _must_ use the release builds! The optimizations here are required
 > as in debug mode, it is expected that nodes are not able to run fast enough to produce blocks.
 
-You will now have two runtimes built in `target/release/` dir:
+You will now have the `tangle-standalone` binary built in `target/release/` dir
 
-1. `tangle-parachain`: Parachain node.
-2. `tangle-standalone`: Standalone node, used in the current standalone Tangle network.
+#### Feature Flags
+
+Some features of tangle node are setup behind feature flags, to enable these features you will have to build the binary with these flags enabled
+
+1. `txpool`
+
+This feature flag is useful to help trace and debug evm transactions on the chain, you should build node with this flag if you intend to use the node for any evm transaction following
+
+```bash
+cargo build --release --features txpool
+```
+
+2. `relayer`
+
+This feature flag is used to start the embedded tx relayer with tangle node, you should build node with this flag if you intend to use the node for any evm transaction following
```

Comment:
you should build a node with this flag if you intend to run a node with a relayer which can be used for  transaction relaying or data querying